src/Controller/InspectController.php

```diff
@@ -59,6 +60,7 @@ public function classes(ContainerInterface $container): ResponseInterface
             fn (string $class) => !str_starts_with($class, 'Yiisoft\\Yii\\Debug\\'),
             fn (string $class) => !str_starts_with($class, 'Yiisoft\\ErrorHandler\\ErrorHandler'),
             fn (string $class) => !str_contains($class, '@anonymous'),
+            fn (string $class) => !in_array(Throwable::class, class_implements($class), true),
```

Comment:
```suggestion
            fn (string $class) => !$class instanceof Throwable,
```

Reply:
$class is a string. `instanceof` cannot be applied for strings.